plugins/plugin_loader.cpp

```diff
@@ -113,14 +140,15 @@ void PluginLoader::NativeLibraryHandleDeleter::operator()(NativeLibraryHandle ha
 {
     if (handle != nullptr && loader != nullptr)
     {
-        if (loader->Free(handle))
+        absl::Status free_status = loader->Free(handle);
+        if (free_status.ok())
```

Comment:
nit: can use init-statement syntax here
```suggestion
        if (absl::Status free_status = loader->Free(handle); free_status.ok())
```

Reply:
fixed